test/e2e/network_test.go

```diff
@@ -36,6 +38,9 @@ func testNetwork(t *testing.T) {
 	err = testCtx.waitForWindowsNodes(gc.numberOfBYOHNodes, false, false, true)
```

Comment:
In `waitForWindowsNodes` we are checking if the [annotations](https://github.com/openshift/windows-machine-config-operator/blob/c7090d62ed3af48c5114a49324639a3639b71781/test/e2e/create_test.go#L298) are present on the node. I was under the impression that if these annotations are present, the HNS networks have been created. Are you saying that even after the node is `Ready`, these networks have not been created? Because that is what your `waitForWindowsNetwork()` is implying.

Reply:
> Are you saying that even after the node is Ready, these networks have not been created?
Good question. That's correct, when the node is Ready, both HNS networks (BaseOVNKubernetesHybridOverlayNetwork and OVNKubernetesHybridOverlayNetwork) were created.

This WIP draft PR is for testing purposes only, I'm not planning to get it merged.

Refer to [WINC-66 comment](https://issues.redhat.com/browse/WINC-666?focusedCommentId=19412173&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-19412173) for details.

Reply:
Nice comment, @jrvaldes however it still does not answer my question. I am not able to understand what scenario you were trying to test with this WIP PR.